python/CMakeLists.txt

```diff
@@ -60,31 +82,17 @@ declare_mlir_python_extension(TorchMLIRPythonExtensions.Main
 # Lazy Tensor Core
 ################################################################################
 
-add_subdirectory(torch_mlir/csrc/base_lazy_backend)
+if(TORCH_MLIR_ENABLE_LTC)
+  add_subdirectory(torch_mlir/csrc/base_lazy_backend)
+endif()
+# Reference backend has a separate check for TORCH_MLIR_ENABLE_LTC.
```

Comment:
why have the separate check? Why not consolidate the check here and just not add the subdirectory if not enabled?

Reply:
I feel like it's cleaner this way, because we need to pull some variables specific to the reference backend for that to work